changelog/issue-1062.md

```diff
@@ -0,0 +1,4 @@
+level: minor
+reference: issue #1062
+---
+The taskcluster cli `rerun` action now takes a `--force` option. It will refuse to rerun non-exception, non-failed tasks without `--force`.
```

Comment:
++!

Reply:
Oh, in fact the `#` here is treated as a YAML comment -- that should be `issue 1062`.  I'll fix in master.